apps/docs/content/guides/database/replication/manual-replication.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Manual Replication
+subtitle: Set up your own replication pipeline for more control
+---
+
+Postgres comes with built-in support for replication. If `pg_replicate` doesn't provide a destination that you need, or if you need more control over the replication process, you can set up your own replication pipeline using Postgres primitives.
```

Comment:
Do we want to have a section on manual replication? I feel like this might confuse our users. Maybe we could just have a small footnote on ETL that we are always looking to add new destinations but if you need a solution quickly you can build your pipeline on top of subscribers offered by Postgres.

Reply:
Agree, this section doesn't sound like as useful we can omit it.